action.yml

```diff
@@ -50,21 +60,28 @@ runs:
           "$EXEC_INSTALL_SCRIPTS" \
           "$DEBUG" \
           "$PACKAGES"
-        echo "CACHE_KEY=$(cat ~/cache-apt-pkgs/cache_key.md5)" >> $GITHUB_ENV
+        if [ -f ~/cache-apt-pkgs/cache_key.md5 ]; then

```

Comment:
This could cause problems since there is metadata attached that ensures the correct cache is loaded based on `${packages} @ ${version} ${force_update_inc}`. In case we decide to change the future behavior. It seems like it would be a no-op to just leave this line as-is.

Reply:
Do you mean `cache_key.md5` should always be generated even when `packages` is empty and `CACHE_KEY` should still be set in that case, as in the old code? Or, should we use some a new `output` variable to indicate that the subsequent steps are skipped?